### ISSUE TITLE
Pale Moon Portable requires updates

### ISSUE THREAD

**Issue by JustOff** [MoonchildProductions/Pale-Moon#1319]
Palemoon-Portable.ini requires the following updates:

1. Default preferences file name was changed (see https://github.com/MoonchildProductions/Pale-Moon/commit/997385020462f91c15230b54c6bfe02e4a62f330)
```
- [WriteToPref=$ExeDir$\defaults\pref\firefox.js]
+ [WriteToPref=$ExeDir$\defaults\pref\palemoon.js]
```

2. Passwords are stored in logins.json since Firefox 32 (see [bug #853549](https://bugzilla.mozilla.org/show_bug.cgi?id=853549))
```
- FileCopy=%MOZ_PROFILE_PATH%\signons.sqlite|$Backup$\$AppName$|o
+ FileCopy=%MOZ_PROFILE_PATH%\logins.json|$Backup$\$AppName$|o
```

**Comment by wolfbeast** [comment]
Absolutely right.

**Comment by wolfbeast** [comment]
Fixed in 27.4.2.1 OOB portable release.
